package.json

```diff
@@ -132,7 +132,7 @@
   "scripts": {
     "clean": "aegir clean",
     "lint": "aegir lint",
-    "dep-check": "aegir dep-check",
+    "dep-check": "aegir dep-check -i protons",
```

Comment:
Are we going to have to do this across the repos? Because I think a solution like https://github.com/ipfs/aegir/issues/1210 would be more sustainable